general-javascript-best-practices.md

```diff
@@ -149,6 +149,27 @@ Use instead of underscore, due to correct semantic versioning, better performanc
 
 Please only require the methods you need rather than the whole library in order to keep build sizes at a minimum.
 
+**Good:**
+```javascript
+const _ = {
+  includes: require('lodash/collection/includes')
+};
+```
+
+**Bad:**
+```javascript
+const _ = require('lodash');
+```
+
+As an alternative (and where Lodash is not currently a dependency) you could install and require each Lodash package individually:
+
+**Better:**
```

Comment:
I disagree with this being the "better" option. I think this is only better if you're using a version of lodash that doesn't let you `require('lodash/func')`. In v4 for example you can just do `require('lodash/includes')`. There is no point adding extra multiple dependencies when we can just install it all in one go. 

I think we should always use the *good* option. It's probably worth adding that when using the *good* option you can't use `_.chain`. Or at least i'm yet to find a way of using it in that way.